includes/class-content-toc.php

```diff
@@ -38,26 +38,7 @@ protected function __construct() {
 		// TOC shortcode counter
 		$this->id_counter = 0;
```

Comment:
If the value is static (doesn't need to be generated) then it should be initialised at the same time as it's declared at the top of the class.